### DISCUSSION TITLE
Eslint editor.codeActionsOnSave so slow

### DISCUSSION THREAD

**Discussion by ivanq3w** [antfu/vitesse#157] [General]
It takes up to 3 seconds to save file...

```
// in .vscode/settings.json
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true,
  },
  ```
<img width="475" alt="Screenshot 2021-07-24 at 15 50 25" src="https://user-images.githubusercontent.com/58051684/126868981-8147cc50-2856-4405-b1fc-1733ae3ebf05.png">

**Comment by ctholho** [comment]
Hi,
I'm using this setup and I have no problems saving.

I don't think you're at the right address for this kind of problem. Look at the information provided by [eslint vs code plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint&ssr=false#overview) to see what might cause your problems. **If you open an issue there please consider providing some more information about your environment and also a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example).**

**Reply by ivanq3w** [reply]
I think its Volar issue
